pkg/values/int.go

```diff
@@ -90,5 +91,23 @@ func (i *Int64) UnwrapTo(to any) error {
 		return nil
 	}
 
+	rv := reflect.ValueOf(to)
+	if rv.Kind() == reflect.Ptr {
```

Comment:
i don't see any obvious new tests for this. is it covered?

Reply:
yeah, the test for the equivalent type should cover this because it uses int.